src/spec/EngineSpec.ts

```diff
@@ -121,4 +121,23 @@ describe('The engine', () => {
      expect(game.canvas.style.top).toEqual('1px');
    });
    
+   it('should accept backgroundColor', () => {
+      var game = new ex.Engine({
+        height: 600,
+        width: 800,
+        backgroundColor: ex.Color.White
+      });
+
+      expect(game.backgroundColor).toEqual(ex.Color.White);
```

Comment:
Because we pass a clone, this will fail I expect. If it doesn't, ignore this. You may need to verify the string representation `toString()` or the individual color channels.

`expect(game.backgroundColor.toString()).toEqual(ex.Color.White.toString());